Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Script_file_added_to_startup-related_Registry_keys.yml

```diff
@@ -0,0 +1,2980 @@
+id: silent-Script file added to startup-related Registry keys
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: silent-Script file added to startup-related Registry keys
+issilent: true
+description: |-
+  This playbook is designed to handle "Script file added to startup-related Registry keys" alerts.
+
+  Playbook Stages:
+
+  Analysis:
+  - Extract the script path from the registry key value.
+  - Check the reputation of the associated processes.
+
+  Investigation:
+
+  - Searches for related XSIAM alerts to identify potential attack patterns.
+  - Retrieve the script file and relevant details for investigation.
```

Comment:
we need to have the same verb form - so if the first is Searches, the next should be Retrieves (not Retrieve) and then Analyzes, Detonates and analyzes (just has to be consistent)